provisioning/service/tests/Config/AttestationMechanismTests.cs

```diff
@@ -120,10 +120,10 @@ public void AttestationMechanismConstructorSucceedOnX509Attestation()
             Assert.AreEqual(AttestationMechanismType.X509, attestationMechanism.Type);
         }
 
-        /* SRS_ATTESTATION_MECHANISM_21_012: [If the type is not `X509` or `TPM`, the getAttestation shall throw ProvisioningServiceClientException.] */
+        /* SRS_ATTESTATION_MECHANISM_21_012: [If the type is not `X509` or `TPM`, the getAttestation shall throw DeviceProvisioningServiceException.] */
```

Comment:
You can just delete these style comments.

Reply:
Do you mean all the comments starting with "SRS_ATTESTATION_MECHANISM" here or just the ones involving "DeviceProvisioningServiceException"?

Reply:
/* SRS_... */